src/cli/bunx_command.zig

```diff
@@ -353,22 +355,28 @@ pub const BunxCommand = struct {
 
         create_package_json: {
             // create package.json, but only if it doesn't exist
-            var package_json = bunx_install_dir.createFileZ("package.json", .{ .truncate = false }) catch break :create_package_json;
+            var package_json = bunx_install_dir.createFileZ("package.json", .{ .truncate = true }) catch break :create_package_json;
             defer package_json.close();
             package_json.writeAll("{}\n") catch {};
         }
 
         var args_buf = [_]string{
-            try std.fs.selfExePathAlloc(ctx.allocator), "add", "--no-summary", try std.fmt.allocPrint(
+            try std.fs.selfExePathAlloc(ctx.allocator), "add", "--no-summary",
+            try std.fmt.allocPrint(
                 ctx.allocator,
                 "{s}@{s}",
                 .{
                     update_request.name,
                     display_version,
                 },
             ),
+            // disable the manifest cache when a tag is specified
+            // so that @latest is fetched from the registry
```

Comment:
IMO you should still cache if it's @{specific-version}, just not for tagged releases

Reply:
`dist_tag` refers to non-ascii npm tagged versions, like `@1.0.0` would be cached but `@latest` or `@next` would not be cached

Reply:
Couldn't we fetch the version tag for `@latest` and check that in the cache?

Reply:
this is basically like 

package.json:
```
{"dependencies": {"create-t3-app": "latest"}}
```

Then, running:
```
bun update
```

`--no-cache` says "ignore Cache-Control headers from npm when deciding whether to check for new versions of dependencies"